File: tags/Versión Final Diciembre/Cliente/Cliente/Validaciones/Obligatorio.cs
using System.Windows.Controls;

namespace Trascend.Bolet.Cliente.Validaciones
{
    public class Obligatorio : ValidationRule
    {
        public override ValidationResult Validate(object value, System.Globalization.CultureInfo cultureInfo)
        {
            string valor = (string) value;

            if (string.IsNullOrEmpty(valor))
                return new ValidationResult(false, Recursos.MensajesValidaciones.Obligarotio);
            else
                return ValidationResult.ValidResult;
        }
    }
}

File: src/Cliente/Cliente_Fac/Validaciones/Obligatorio.vb
Imports System.Windows.Controls

Namespace Validaciones
    Public Class Obligatorio
        Inherits ValidationRule
        Public Overrides Function Validate(ByVal value As Object, ByVal cultureInfo As System.Globalization.CultureInfo) As ValidationResult
            Dim valor As String = DirectCast(value, String)

            If String.IsNullOrEmpty(valor) Then
                Return New ValidationResult(False, Recursos.MensajesValidaciones.Obligarotio)
            Else
                Return ValidationResult.ValidResult
            End If
        End Function
    End Class
End Namespace